torch/_inductor/compile_fx.py

```diff
@@ -762,6 +763,7 @@ def codegen_and_compile(
                 local=config.fx_graph_cache,
                 remote=fx_graph_remote_cache,
             )
+            set_feature_use("//pytorch/remote_cache:fx_graph_memcache_version", True)
```

Comment:
Isn't this JK an integer value? I guess it doesn't make sense to log that integer though since we really want to know if the higher-level check passes. But then the name of the setting is kind of confusing. Eh, whatever; I guess there's no ambiguity.

But Also: I guess `load` / `codegen_and_compile` could raise, in which case we wouldn't have the setting. Maybe we want to set the feature first?

Reply:
Moved upwards so it'll always log. If you feel strongly I can drop the JK naming scheme and just use "normal names" - i.e. inductor_fx_cache or something.